_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -112,7 +113,35 @@ class HealthCareApp extends React.Component {
           },
           childInformation: {},
           annualIncome: {},
-          deductibleExpenses: {}
+          deductibleExpenses: {},
+        },
+
+        // TODO: insuranceInformation should be an array where each row
+        //       contains all the following fields:
+        insuranceInformation: {
+          insuranceInfo: {
+            isCoveredByHealthInsurance: false,
+            insuranceName: undefined,
```

Comment:
use empty-string like `''` instead of undefined.

Undefined, in react, will act as if the property was not even set.